.github/workflows/ci.yaml

```diff
@@ -0,0 +1,38 @@
+name: ci
```

Comment:
I suggest you rename this file to `docker.yaml` and this line to `name: docker` because there are other GitHub actions CI coming.

Reply:
Ok for `docker.yaml` 👍 (maybe `docker-ci.yaml` or `ci-docker.yaml` in order to quickly see CI workflow files ?)

For the `name` I am going to check something on my side and get back to you

Reply:
OK so I suggest we don't change the `name` entry and set it to `ci` in each workflow file related to a CI workflow.

Here is an example of what happens when we have two workflow files (ex: `ci-docker.yaml` & `ci-lint.yaml`) having the `name` entry set to `ci`:

<img width="1680" alt="Screenshot 2021-01-13 at 09 58 20" src="https://user-images.githubusercontent.com/11849873/104429560-099a0700-5586-11eb-84e5-7968033607e2.png">

We can see that each check name is in the form of `ci/<jobName>` 👍

Reply:
Let me know about the fileName 👍 

- docker.yaml
- ci-docker.yaml
- something-else ...

Reply:
we can go with anything which contains the word `docker`

Reply:
Done 👍